tyk-headless/values.yaml

```diff
@@ -53,7 +53,7 @@ gateway:
     enabled: false
     annotations: {}
     # kubernetes.io/ingress.class: nginx
-    # kubernetes.io/tls-acme: "true"
+    # kubernetes.io/tls-acme: true
```

Comment:
You do need to quote bools for the annotations — otherwise the value will be misinterpreted (same in the other annotations). Again, because dynamic typing and only strings are supported in the annotations, see e.g. https://github.com/kubernetes/ingress-nginx/issues/1990

Not sure if `tls-acme` is still relevant at all (`kube-lego` is long deprecated) though.

Reply:
Ahhh good catch I will throw these back in.